id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/manager/CredentialStatusManager.java

```diff
@@ -215,6 +222,7 @@ public void handleRemoveIdStatusEvent(EventModel eventModel) {
 					"handleRemoveIdStatusEvent", "inside handleRemoveIdStatusEvent()");
 			String idHash = (String) eventModel.getEvent().getData().get(ID_HASH);
 			handleRepo.deleteByHandleHash(idHash);
+			auditHelper.audit(ID_REPO_CORE_SERVICE, REMOVE_ID_STATUS, idHash, ID, "Remove ID Status Event");
```

Comment:
1. In case of VID and UIN revoke we get the acknowledgment? 
2. We should check if auditing is enough, or should we maintain the history table, lets check, if required we shall take it as the next sprint task.